### PR TITLE
chore: Update test and script after app-manager renaming

### PR DESCRIPTION
**Pull request by benjaminParisel**
apps-manager-doc repository is renamed to [bonita-central-doc](https://github.com/bonitasoft/bonita-central-doc/pull/5)

This PR is needed to allow the preview deploy in content-repository.

### CODE REVIEW

File: test/documentation-content/bonita-central/1.0/antora.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-name: apps-manager
-title: Apps Manager
+name: bonita-central
```
Comment (tbouffard) [inline test/documentation-content/bonita-central/1.0/antora.yml:1]:
```suggestion
name: central
```

File: test/documentation-content/bonita-central/1.0/antora.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+title: Bonita Central
```
Comment (tbouffard) [inline test/documentation-content/bonita-central/1.0/antora.yml:2]:
```suggestion
title: Central
```

**Review by tbouffard — COMMENTED** [review]
Paths in the test directories should be "central" instead of "bonita-central" for consistency with the component name (the preview script should be updated accordingly).
I will test locally with the test data later today

**Comment by benjaminParisel** [comment]
@tbouffard I have integrated yours comments, i will merged it and rerun the preview deploy on bonita-central PR

**Review by jeromecambon — APPROVED** [review]
LGTM, although 'Central' for the component seems a bit ambiguous to me.

**Pull request merged**